worlds/yachtdice/Options.py

```diff
@@ -0,0 +1,331 @@
+    Extra progression items are added that will help you on your quest.
+    You can set the weight for each extra progressive item in the following options.
+
+    Of course, more dice = more points!
+    """
+
+    display_name = "Weight of adding Dice"
+    range_start = 0
+    range_end = 100
+    default = 5
+
+
+class ChanceOfRoll(Range):
+    """
+    With more rolls, you will be able to reach higher scores.
+    """
+
+    display_name = "Weight of adding Roll"
+    range_start = 0
+    range_end = 100
+    default = 20
+
+
+class ChanceOfFixedScoreMultiplier(Range):
+    """
+    Getting a Fixed Score Multiplier will boost all future scores by 10%.
+    """
+
+    display_name = "Weight of adding Fixed Score Multiplier"
+    range_start = 0
+    range_end = 100
+    default = 30
+
+
+class ChanceOfStepScoreMultiplier(Range):
+    """
+    The Step Score Multiplier boosts your multiplier after every roll by 1%, and resets on sheet reset.
+    So, keep high scoring categories for later to get the most out of them.
+    By default, this item is not included. It is fun however, you just need to know the above strategy.
+    """
+
+    display_name = "Weight of adding Step Score Multiplier"
+    range_start = 0
+    range_end = 100
+    default = 0
+
+
+class ChanceOfDoubleCategory(Range):
+    """
+    This option allows categories to appear multiple times.
+    Each time you get a category after the first, its score value gets doubled.
+    """
+
+    display_name = "Weight of adding Category copy"
+    range_start = 0
+    range_end = 100
+    default = 50
+
+
+class ChanceOfPoints(Range):
+    """
+    Getting points gives you... points. You can get 1 point, 10 points, and even 100 points.
+    """
+
+    display_name = "Weight of adding Points"
+    range_start = 0
+    range_end = 100
+    default = 20
+
+
+class PointsSize(Choice):
+    """
+    If you choose to add points to the item pool, do you prefer many small points,
+    medium-size points, a few larger points, or a mix of them?
+    """
+
+    display_name = "Size of points"
+    option_small = 1
+    option_medium = 2
+    option_large = 3
+    option_mix = 4
+    default = 2
+
+
+class MinimizeExtraItems(Choice):
+    """
+    Besides necessary items, Yacht Dice has extra useful/filler items in the item pool.
+    It is possible however to decrease the number of locations and extra items.
+    This option will:
+    - decrease the number of locations at the start (you'll start with 2 dice and 2 rolls).
+    - will limit the number of dice/roll fragments per dice/roll to 2.
+    - in multiplayer games, it will reduce the number of filler items.
+    """
+
+    display_name = "Minimize extra items"
+    option_no_dont = 1
+    option_yes_please = 2
+    default = 1
+
+
+class AddExtraPoints(Choice):
+    """
+    Yacht Dice typically has space for extra items.
+    If there is space, would you like bonus points shuffled in the item pool?
```

Comment:
```
    If there is space, would you like bonus points shuffled in the item pool?
```
Having a question in the description feels odd. I would potentially rephrase it like "you can add bonus points to the item pool" instead of a question